src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -62,6 +64,30 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         #region Authorization and Credentials
 
         /// <summary>
+        /// gets or sets the AllowUnencryptedAuthorization property
+        /// </summary>
+        [Parameter]
+        public virtual SwitchParameter AllowUnencryptedAuthorization { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Authorization property used to determin the Authorization method for the web session.!--
```

Comment:
A more detailed description is welcomed.
Please add about correlation with other parameters from PR description.

Reply:
fixed.